sdk/tools/spec2def/spec2def.c

```diff
@@ -950,15 +1082,13 @@ ParseFile(char* pcStart, FILE *fileDest, PFNOUTLINE OutputLine)
             /* Go to next token */
             if (!(pc = NextToken(pc)))
             {
-                fprintf(stderr, "%s line %d: error: expected token\n", pszSourceFileName, nLine);
-                return -13;
+                Fatal(pszSourceFileName, nLine, pcLine, pc, 1, "Unexpected end of line");
             }
 
             /* Verify syntax */
             if (*pc++ != '(')
             {
-                fprintf(stderr, "%s line %d: error: expected '('\n", pszSourceFileName, nLine);
-                return -14;
+                Fatal(pszSourceFileName, nLine, pcLine, pc, 0, "Expected '('");
```

Comment:
I assumed you'd want `pc - 1` and `1` here.

Reply:
-1 is right, but 0 is fine, as it will mark the next token.